### ISSUE TITLE
We need the capability to launch validation set evaluations on multiple datasets (supporting clip-zero-shot and linear probing evaluations during training time)

### ISSUE THREAD

**Issue by Oufattole** [Oufattole/meds-torch#148]
See an example of how to load multiple datasets here:
https://github.com/Oufattole/meds-torch/blob/8f023b553222abfe1ad9f731d18125e02134e01a/src/meds_torch/configs/data/bimodal_pytorch_dataset.yaml#L57

This uses hydra templates, maybe there is a simpler way to allow users to do this in the experiment_mtr.

**Comment by Oufattole** [comment]
Here is the training recipe for this bimodal model: https://github.com/Oufattole/meds-torch/blob/8f023b553222abfe1ad9f731d18125e02134e01a/MIMICIV_INDUCTIVE_EXPERIMENTS/configs/meds-torch-configs/experiment/triplet_bimodal_mtr.yaml#L56

Here is the bicl yaml that allows the multiple datasets to be passed: https://github.com/Oufattole/meds-torch/blob/indexable_multimodal/src/meds_torch/configs/model/bicl.yaml

via the attribute:
```yaml
task_names: [
  "${data.eval_datasets[1].task_name}", 
  "${data.eval_datasets[2].task_name}",
]
```
